dipy/data/__init__.py

```diff
@@ -39,7 +39,9 @@
                                fetch_cenir_multib,
                                read_cenir_multib,
                                fetch_mni_template,
-                               read_mni_template)
+                               read_mni_template,
+                               fetch_ivim,
```

Comment:
Is this rebased on top of master? I get an error when I try to run the example from this branch, because it can't find fetch_ivim.